TestSwitchApi/TestSwitchApi/Models/DataModels/AdminUserDataModel.cs

```diff
@@ -8,7 +8,13 @@ public class AdminUserDataModel
         [Key]
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Id { get; set; }
-        public string Email { get; set; }
+
+        public string Email
+        {
+            get => Email;
+            set => Email.ToLower();
```

Comment:
Does this work? I was expecting you to need to use `set => value.ToLower`.

Reply:
somehow an old commit has sneaked in here